### ISSUE TITLE
Switch back to `node-wpapi`?

### ISSUE THREAD

**Issue by ryelle** [ryelle/wordpress-query-posts#5]
¯\_(ツ)_/¯

Need to actually play around with an isomorphic app.

**Comment by SastraNababan** [comment]
I'm curious why it use wordpress-rest-api-oauth-1 since v1.1.0 instead of node-wpapi, do you have plan to switch it back to node-wpapi on next version ?

**Comment by ryelle** [comment]
@SastraNababan At the time, it was because the API was in beta, and some endpoints were changing (like the `filter` parameter, which was removed). node-wpapi didn't update immediately, and had a much more custom implementation. wordpress-rest-api just let me pass through a URL + query params directly, so I was able to react quickly in case the endpoints changed. It was also easier to test responses using something like [Insomnia](https://insomnia.rest/).

Do you have a specific use case where node-wpapi is necessary?

**Comment by SastraNababan** [comment]
I have a few issues with react native. Should I submit new issue ?

**Comment by ryelle** [comment]
Yeah, new issues for any problems you've encountered would be great. Thanks 🙂
